src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,41 @@
+package tasks;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * constructor of Task.
+     *
+     * @param description description of the task
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+
+    public String getStatusIcon() {
+        return (isDone ? "+" : "-"); //return tick or X symbols
```

Comment:
```suggestion
        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```
I suggest using actual tick and X symbol with the specified code.

Reply:
thanks for your suggestion! I initially used ticks but due to character encoding difference, intellij console is not able to properly show the special chars, which is why I later change to +/-. But it's true that tick/x look nicer.